lib/screens/tree_list/tree_list_view_model.dart

```diff
@@ -21,24 +24,29 @@ abstract class TreeListViewModelBase with Store, ViewModel {
       DependecyInjection.instance.get<TreeStoreBase>();
 
   final GetTreeList _useCase = DependecyInjection.instance.get<GetTreeList>();
+  final ConnectionStatusManager connectionStatusManager =
+      DependecyInjection.instance.get<ConnectionStatusManager>();
 
   @observable
   bool isLoadingTrees = false;
+  bool hasInternetConnection = false;
+  List<Tree> newTrees = [];
 
   @override
   void init() {
-    if (treeStore.isTreeListEmpty()) {
-      fetch();
-    }
+   activateConnectionManager();
```

Comment:
Do not activate the manager here. It's already been activated in the main_common 
And why delete the initial fetch ?